server.go

```diff
@@ -296,25 +304,79 @@ type Server struct {
 	// DecorateWriter is optional, allows customization of the process that writes raw DNS messages.
 	DecorateWriter DecorateWriter
 
+	// UDP packet or TCP connection queue
+	queue chan *response
+	// Workers count
+	workersCount int32
 	// Shutdown handling
 	lock    sync.RWMutex
 	started bool
 }
 
+func (srv *Server) worker(w *response) {
+	defer atomic.AddInt32(&srv.workersCount, -1)
+
+	srv.serve(w)
+
+	count := 0
+	ticker := time.NewTicker(idleWorkerTimeout)
```

Comment:
This should be `time.NewTimer` with `ticker.Reset` below. `time.NewTicker` will drop sends on the floor.

Reply:
Done